submissions/vovan-zt/Memory-Pair-Game/script.js

```diff
@@ -0,0 +1,128 @@
+﻿'use strict'
+
+const cards = [
+    { name: 'asuma', image: 'img/13.jpg' },
+    { name: 'hinata', image: 'img/14.jpg' },
+    { name: 'naruto', image: 'img/15.jpg' },
+    { name: 'gaara', image: 'img/16.jpg' },
+    { name: 'kakashi', image: 'img/17.jpg' },
+    { name: 'yamato', image: 'img/18.jpg' },
+]
+
+cards.push(...cards)
+
+const sortingCards = (dataBase) => {
```

Comment:
You don't have "dataBase" in your app :)

Also, it will be `sortCards`. Or even `shuffle`